vmb/matrimony/views.py

```diff
@@ -1,3 +1,44 @@
-from django.shortcuts import render
+from django.shortcuts import render, get_object_or_404
+from vmb.matrimony.models.profiles import MatrimonyProfile
+from vmb.matrimony.models import Match
 
 # Create your views here.
+def response(request, id):
```

Comment:
The name does not make sense. If we want to have dedicated views for handling `match` response, we should at least call it `match_response`.

IMO, if we are not implementing AJAX to update match response from the `match_detail` page, it's not worth adding a separate view for response and templates for it.